### PR TITLE
Inactivity timeout & Ticker Animation

### PR DESCRIPTION
**Pull request by abdulrauf11**
- [x] Implemented inactivity timeout on Quiz level by using [react-idle-timer](https://idletimer.dev/). You can control it through the `appConfig.js` file. 
- [x] Implemented ticker animation on Attract Screen.

### Preview Link
https://ctsc-career-quiz-git-timeranimation-rauf-tech.vercel.app?_vercel_share=WNgKnzTQPrBlP4471BMsRKZ7DQOviFtG

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ module.exports = {
...
       },
+
+      keyframes: {
+        ticker: {
+          '0%': { transform: 'translateX(0)' },
+          '100%': { transform: 'translateX(-100%)' },
+        },
+      },
+      animation: {
+        ticker: 'ticker 15s linear infinite',
```
Comment (tnordberg) [inline tailwind.config.js]: Please slow this down more. Change 15s to 22s.

**Review by tnordberg — COMMENTED** [review]
Thanks @abdulrauf11 - I noticed the profile image crops have uneven padding, which is making the carousel spacing look a little off. (https://d.pr/i/Nssuhn)

If my team adjusts those crops so the circles are perfectly centered with equal left-right padding, that should fix the spacing and alignment, right?

If that's the case, I'll approve this after you slow down the animation a little. My team can fix those images later this week.

**Comment by abdulrauf11** [comment]
> Thanks @abdulrauf11 - I noticed the profile image crops have uneven padding, which is making the carousel spacing look a little off. (https://d.pr/i/Nssuhn)
> 
> If my team adjusts those crops so the circles are perfectly centered with equal left-right padding, that should fix the spacing and alignment, right?
> 
> If that's the case, I'll approve this after you slow down the animation a little. My team can fix those images later this week.

@tnordberg that is correct! The uneven padding is because of the different image sizes and padding around them.
You may also need to alter the previous image icons as well.

File: tailwind.config.js (hunk 1 above)
Comment (abdulrauf11) [reply tailwind.config.js]: Fixed.

**Review by tnordberg — APPROVED** [review]

**Comment by tnordberg** [comment]
@abdulrauf11 could you please merge this approved PR?

**Pull request merged**